bulk_indexer.go

```diff
@@ -132,6 +139,8 @@ type BulkIndexerResponseItem struct {
 		Type   string `json:"type"`
 		Reason string `json:"reason"`
 	} `json:"error,omitempty"`
+
+	Source string `json:"source"`
```

Comment:
```suggestion
	Source string `json:"-"`
```

This doesn't come back from Elasticsearch, let's make that clear with `json:"-"`